### PR TITLE
moved all_posts to index, all_posts_rss to rss, updated README with n…

### PR DESCRIPTION
**Pull request by jywarren**
…ew non-bower info

Whoops, comment too long. 

So this is not tested successfully, as I'm trying to learn how to run tests properly in https://github.com/zenhack/iron-blogger2/issues/28. But I believe it's correct, for what it's worth.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,10 +32,7 @@ for the database, and run `ironblogger init-db`
 
 ## Static assets
 
-Static assets are managed via Bower (http://bower.io) and can be installed with the command `bower install`. This helps keep our standard css/js/font libraries orderly. 
-
-Bower itself can be installed with `npm install bower`
-
+Static assets have been submoduled and are kept in `ironblogger/static/bower_components`, which used to be managed by Bower, which is why it has that name.
```
Comment (zenhack) [inline README.md]: Really we ought to just rename the directory... not going to block this patch on it though; that can certainly come later.

File: ironblogger/templates/layout.html [hunk 2]
```diff
@@ -26,11 +26,10 @@
 					{% set targ, text, method = ('do_logout', 'Log Out', 'post') %}
 					{% endif %}
 					<ul class="nav nav-justified">
-						<li><a href="{{ url_for('show_all_posts') }}">Posts</a></li>
```
Comment (zenhack) [inline ironblogger/templates/layout.html]: We should probably still have a nav item for the front page (maybe home? maybe keep calling it posts? not sure); the fact that the site's top level header is a link to the front page probably isn't obvious enough given that there's actually functionality there now.

**Comment by zenhack** [comment]
@jywarren the pagination patch I pushed yesterday seems to have caused some merge conflicts. When you get back to this, you'll have to fix those --- I don't think they'll be bad.

**Comment by jywarren** [comment]
OK, I rebased, and everything seems to work, although it was a a bit fraught. Two things:

1. Can you check that /status works? I may just not have run it, but my local copy doesn't show any content. 
2. I wasn't sure about totally removing /posts, since that seems like a reasonable (future) REST resource route. So I basically have index and posts (changed from all-posts, with the thought of REST-friendly-routing nomenclature) doing the same thing, which isn't super DRY. But my python is a bit rusty, so I wasn't sure the best practice way to DRY it up; i'm happy to make a change if you can suggest it. My thought was to have both routes lead to the same controller code, and to re-use the template, but I kind of want to defer to you on this.

**Comment by jywarren** [comment]
Oops, i had missed some of your inline comments, but now they're addressed. I also ended up DRYing up the code a bit and it now seems OK to me -- just double-check how I went about that. For example, there's no longer an index template, and the index route just runs the show_posts method.

**Comment by zenhack** [comment]
`/status` works here. Probably you just didn't import the bloggers.yaml and/or run `sync-posts`/`assign-rounds`. See the README.

I think having a redirect would be better than just calling the function internally; this way the fact that it's the same page is visible to things like search engines crawling the site.

http://flask.pocoo.org/docs/0.10/quickstart/#redirects-and-errors

Everything else looks good.

**Comment by jywarren** [comment]
OK! There we are.

**Comment by zenhack** [comment]
Merged and deployed! Thanks again.

**Pull request merged**